src/services/permissions/checks/domains/interventions/InterventionsCheck.ts

```diff
@@ -1,9 +1,9 @@
-import PermissionsCheckRequest from '../../PermissionsCheckRequest'
+import PrisonerPermissionsContext from '../../../../../types/internal/permissions/PrisonerPermissionsContext'
 import { InterventionsDomainPermissions } from '../../../../../types/public/permissions/domains/interventions/InterventionsDomainPermissions'
 import personInterventionsCheck from './personInterventions/PersonInterventionsCheck'
 
-export default function interventionsCheck(request: PermissionsCheckRequest): InterventionsDomainPermissions {
+export default function interventionsCheck(context: PrisonerPermissionsContext): InterventionsDomainPermissions {
```

Comment:
Renamed `request` to `context`, seemed a more accurate description of that object being passed around